setup.py

```diff
@@ -1,11 +1,27 @@
-from setuptools import setup
+from setuptools import setup, find_packages
+
+with open('README.md', 'r', encoding='utf-8') as f:
+    long_description = f.read()
 
 setup(
-    name='Whist-Core',
+    name='whist-core',
     version='0.0.1',
```

Comment:
```suggestion
    version='0.0.2',
```

there was already a tag uploaded to PyPI with `0.0.1` which was deleted by mistake, therefore the next release should be `0.0.2`